core/common/src/IModel.ts

```diff
@@ -135,6 +135,8 @@ export interface SnapshotOpenOptions extends IModelEncryptionProps, OpenDbKey {
   lazyBlockCache?: boolean;
   /** @internal */
   autoUploadBlocks?: boolean;
+  /** @internal */

```

Comment:
Even though this is internal, we should add a description of what it does.

Reply:
What about the description used for the tempFileBase on the addon side? 

"The "base" name that can be used for creating temporary files related to this Db. The string should be a name related to the current Db filename using
some known pattern so that all files named "baseName*" can be deleted externally during cleanup. It must be the name of a file (that may or may not exist)
in a writable directory. 
If not present, use the db file name."

Maybe change the last line to "If not present, the baseName will default to the database's file name (including the path)."?

Reply:
sounds good to me.

Reply:
Changed it